src/lsp-server.ts

```diff
@@ -50,8 +50,8 @@ export class LspServer {
             return this.options.tsserverPath;
         }
         this.logger.info("Looking up 'tsserver' in " + this.rootPath())
-        const path = findPathToModule(this.rootPath(), 'typescript/bin/tsserver')
-        return path || 'tsserver'
+        let path = findPathToModule(this.rootPath(), '.bin/tsserver') || 'tsserver'
+        return isWindows() ? path + ".cmd" : path
```

Comment:
I still think it might be better to refactor this into a function since cli help also needs this logic.

```typescript
.option('--tsserver-path <path>',
        `Specifiy absolute path to tsserver. example: --tsserver-path=${isWindows() ? 'c:\\tsc\\tsserver.cmd' : '/bin/tsserver'}`,
        isWindows() ? 'tsserver.cmd' : 'tsserver')
```